site/src/examples/date-picker/ControlledOpen.tsx

```diff
@@ -63,7 +63,8 @@ export const ControlledOpen = (): ReactElement => {
   }, []);
 
   const handleOpenChange = useCallback(
-    (newOpen: boolean, _event?: Event, reason?: DatePickerOpenChangeReason) => {
+    (newOpen: boolean, reason?: DatePickerOpenChangeReason) => {
+      // reason === undefined denotes a programmatic/controlled open, so focus behaviour is owned by the code
```

Comment:
comments

Reply:
intentional, explaining here that when reason is undefined, that you own the focus responsibilities.